src/Lorentz/Contracts/BaseDAO/Types.hs

```diff
@@ -167,7 +167,7 @@ instance TypeHasDoc MigrationStatus where
     "Migration status of the contract"
 
 -- | Storage of the FA2 contract
```

Comment:
While we are at it, I think we should at least change "the FA2 contract" to "a DAO contract" (because baseDAO is a group of contracts and this storage is used in all of them). Also I think `contractExtra` parameter should be described, but in proper world `proposalMetadata` should have been described already and it's not described which is not your fault. So feel free to keep this short description if you don't want to expand it, but let's change this "the FA2 contract" thing at least (otherwise nobody will ever do it).

Reply:
>While we are at it, I think we should at least change "the FA2 contract" to "a DAO contract"

:peka: